Node.js/GettingStarted/processcsv.py

```diff
@@ -0,0 +1,100 @@
+import csv
+import json
+from azure.storage.blob import BlockBlobService
+import datetime
+import os
+import argparse
+
+#### TODO: Get the configuration
+
+storage_acc_name = '<storage account name>'
```

Comment:
Consider passing these via environment variables instead of duplicating them in the Python script.  Or pass SAS URLs to the containers instead of requiring the storage credentials to be passed to the compute node at all.